nextcord/ext/help_commands/embedded.py

```diff
@@ -0,0 +1,94 @@
+from typing import Mapping, Union
+
+from nextcord.ext.commands import Cog, Command, Group, HelpCommand
+
+from nextcord import Embed, Message
+
+
+class EmbeddedHelpCommand(HelpCommand):
+    """Custom help command override using embeds"""
+
+    def __init__(self, **options):
+        self.cog_heading = options.pop("cog_heading", "Cog Commands:")
+        self.group_heading = options.pop("group_heading", "Group Commands:")
+        self.command_heading = options.pop("command_heading", "Command:")
+        self.dm_help = options.pop("dm_help", False)
+        self.no_category = options.pop("no_category", "No Category")
+
+        super().__init__(**options)
+
+    @staticmethod
+    def determine_group_or_command(obj: Union[Command, Group]):
+        return f"`{obj.name}[group]`" if isinstance(obj, Group) else f"`{obj.name}`"
+
+    async def send_embed(self, emb: Embed):
+        return (
+            await self.context.author.send(embed=emb)
+            if self.dm_help
+            else await self.context.send(embed=emb)
+        )
+
+    # help
+    async def send_bot_help(self, mapping: Mapping) -> Union[Message, None]:
+        emb = Embed(
+            title=f"**Full command list.** For a detailed guide, check {self.context.clean_prefix}help <name of command>"
+        )
+
+        bare_cmd_list = " ".join(
+            self.determine_group_or_command(cmd) for cmd in self.context.bot.commands if not cmd.cog
+        )
+        emb.add_field(name=self.no_category, value=bare_cmd_list, inline=False)
```

Comment:
This should probably be added conditionally based on whether any commands exist in no category. 

I tried testing this and got this error

![image](https://user-images.githubusercontent.com/20955511/178766361-67936c4e-c2ac-474f-99f6-fd5042376cb5.png)

Reply:
do you think a `if bare_cmd_list != "":` would be fine? I can't really think of much edge cases other than no commands at all

Reply:
Yeah, it's similar to how you did `if cog_cmd_list:`